operations/before.go

```diff
@@ -62,6 +62,16 @@ var (
 		return c.Set(hostFlagName, host)
 	}
 
+	checkCommitMessageFlagFlag = func(c *cli.Context) error {
```

Comment:
nit: because it's not really a flag (i think) people use can we call it something like validateCommitMessage?

Reply:
or `checkExtraArgs` would work as well. 
I'm fine with `checkCommitMessageFlag` because it matches with requirePatchIDFlag, but the extra Flag confuses me a bit.